src/components/alert/alert.scss

```diff
@@ -149,3 +153,10 @@
     transform: scale(scale.$effect-shrink);
   }
 }
+
+.c-alert__error {
+  .c-alert__extra {
+    background-color: var(--theme-color-background-icon-error);
+    color: var(--theme-color-icon-error);
+  }
+}
```

Comment:
Because these styles impact `.c-alert__extra`, they should be located near the `.c-alert__extra` styles.